### PR TITLE
DOC: fix plot submethods whatsnew example

### PR DESCRIPTION
**Pull request by shoyer**
This was not building properly.

### CODE REVIEW

File: doc/source/whatsnew/v0.17.0.txt [hunk 1]
```diff
@@ -150,7 +150,7 @@ The Series and DataFrame ``.plot()`` method allows for customizing :ref:`plot ty
 To alleviate this issue, we have added a new, optional plotting interface, which exposes each kind of plot as a method of the ``.plot`` attribute. Instead of writing ``series.plot(kind=<kind>, ...)``, you can now also use ``series.plot.<kind>(...)``:
 
 .. ipython::
-  :verbatim:
+    :verbatim:
```
Comment (jreback) [inline doc/source/whatsnew/v0.17.0.txt]: doesn't this have to go under the 'i'?

Comment (shoyer) [inline doc/source/whatsnew/v0.17.0.txt]: I think the key thing is aligning it with the indentation below. This version builds properly, at least:
screen shot 2015-09-12 at 4 31 52 pm

Comment (jreback) [inline doc/source/whatsnew/v0.17.0.txt]: ok gr8

**Comment by shoyer** [comment]
OK, not going to wait for Travis on this one because Travis doesn't even bother to build the docs.

File: doc/source/whatsnew/v0.17.0.txt (hunk 1 above)
Comment (jorisvandenbossche) [inline doc/source/whatsnew/v0.17.0.txt]: yes, you can choose whatever indentation you want, as long as it is the same for the code below (most of the time, I find using 4 spaces the easiest as then you can use tab)

**Pull request merged**